ui/carousel.js

```diff
@@ -72,6 +72,7 @@
         });
         scrollable.smoothDivScroll("recalculateScrollableArea");
         scrollable.smoothDivScroll("startAutoScrolling");
+        $(document).trigger('uiTagCarouselImages');
```

Comment:
We should make this change to carousel.coffee as well.

I'd suggest something more specific the the event you are tagging. `uiTagCarouselImages` is a bit generic. 

The story is unclear about which events we're tagging, BTW.